### PR TITLE
jQuery Challenge 5 to 10 French Translation

### PR DESCRIPTION
**Pull request by Tagada85**
5 Challenges jQuery.
I still need to sharpen my git skills. Need to work on branchs and squash stuff I believe.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -864,7 +912,7 @@
         "In addition to moving elements, you can also copy them from one place to another.",
         "jQuery has a function called <code>clone()</code> that makes a copy of an element.",
         "For example, if we wanted to copy <code>target2</code> from our <code>left-well</code> to our <code>right-well</code>, we would use:",
-        "<code>$(\"#target2\").clone().appendTo(\"#right-well\");</code>",
+        "<code>$(\"#target2\").clone().appendTo(\"#right-well\");</code> ",
```
Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/jquery.json]: You should remove this space.

**Comment by atjonathan** [comment]
@Tagada85, please get someone in the [TranslationFrench](https://gitter.im/FreeCodeCamp/TranslationFrench) chat room to peer review this, thanks :+1:

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 2]
```diff
@@ -447,7 +452,15 @@
         "Füge die Klasse <code>animated</code> zu allen Elementen des Typs <code>button</code> hinzu.",
         "Füge die Klasse <code>shake</code> zu allen Buttons mit der Klasse <code>.btn</code> hinzu.",
         "Füge die Klasse <code>btn-primary</code> zu dem Button mit der ID <code>#target1</code> hinzu."
-      ]
+      ],
+       "titleFr" : "Cibler le même élément avec plusieurs sélecteurs jQuery",
+       "descriptionFr" : [
+         "Á présent, vous connaissez trois façons de cibler les éléments: par type: <code>$(\"button\")</code>, par classe: <code>$(\".btn\")</code>, et par id <code>$(\"#target1\")</code>.",
```
Comment (mizujin) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Small typo on accent: " Á présent " should be " À présent"

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 3]
```diff
@@ -586,7 +606,17 @@
         "Das ist ein wenig anders als eine normale CSS-Deklaration, weil die CSS-Eigenschaft und deren Wert in Anführungszeichen sind und durch ein Komma anstatt eines Doppelpunkts getrennt werden.",
         "Lösche deine jQuery Selektoren, damit eine leere <code>Document Ready Funktion</code> übrig bleibt.",
         "Wähle <code>target1</code> und ändere dessen Farbe zu Rot."
-      ]
+      ],
+      "titleFr" : "Changer le CSS d'un élément en utilisant jQuery",
+       "descriptionFr" : [
+         "Nous pouvons aussi changer le CSS d'un élément HTML directement avec jQuery.",
+         "jQuery dispose d'une fonction appelée <code>.css()</code> qui vous permet de changer le CSS d'un élément.",
+         "Voici comment on changerait sa couleur en bleu:",
```
Comment (mizujin) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Would it be better to simply say " Voici comment changer sa couleur en bleu:"

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 4]
```diff
@@ -655,7 +685,25 @@
         "So kannst du alle <code>Buttons</code> deaktivieren:",
         "<code>$(\"button\").prop(\"disabled\", true);</code>",
         "Deaktiviere nur den <code>target1</code> Button."
-      ]
+      ],
+      "titleFr" : "Désactivez un élément en utilisant jQuery",
+       "descriptionFr" : [
+         "Vous pouvez également changer les propriétés non-CSS des éléments HTML avec jQuery. Par exemple, vous pouvez désactiver les boutons.",
+         "Lorsque vous désactivez un bouton, il devient gris et on ne peut plus cliquer dessus.",
+         "jQuery dispose d'un fonction appelée <code>.prop()</code> qui vous permet de modifier les propriétés des éléments.",
```
Comment (mizujin) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Should read " dispose d'un**e** fonction appelée "

**Comment by Tagada85** [comment]
Thanks @mizujin .It is cleaner without the use of the conditional 👍

**Comment by atjonathan** [comment]
### **LGTM :+1:**
We will merge for now and if something is incorrect, it can be changed later.
Thanks for your PR @Tagada85 :tada:

**Pull request merged**